doc/changelog/1.13.0-changelog.rst

```diff
@@ -364,7 +364,7 @@ A total of 309 pull requests were merged for this release.
 * `#8928 <https://github.com/numpy/numpy/pull/8928>`__: BUG: runtests --bench fails on windows
 * `#8929 <https://github.com/numpy/numpy/pull/8929>`__: BENCH: Masked array benchmarks
 * `#8939 <https://github.com/numpy/numpy/pull/8939>`__: DEP: Deprecate `np.ma.MaskedArray.mini`
-* `#8942 <https://github.com/numpy/numpy/pull/8942>`__: DOC: stop refering to 'S' dtype as string
```

Comment:
We may want to leave the changelogs alone to show exactly what the title of the PRs are, misspellings and all.

Reply:
Or we can rename the PRs